schemas/restrictions.js

```diff
@@ -0,0 +1,37 @@
+/*!
+ * Copyright (c) 2021 Digital Bazaar, Inc. All rights reserved.
+ */
+
+export default {
+  restriction: {title: 'restriction',
+    type: 'object',
+    required: ['zone', 'resource', 'method', 'methodOptions'],
+    additionalProperties: false,
+    properties: {
+      zone: {
+        type: 'string'
+      },
+      resource: {
+        type: 'string'
+      },
+      method: {
+        type: 'string',
+        enum: ['limitOverDuration']
+      },
+      methodOptions: {
+        type: 'object',
+        required: ['limit', 'duration'],
+        additionalProperties: false,
+        properties: {
+          limit: {
+            type: 'integer',
+            minimum: 0
```

Comment:
Is there a use case for zero?  Would the minimum possibly be 1?

Reply:
I cannot think of a use case for 0, bumping to 1.